src/contracts/uniswap/UniswapV2Library.sol

```diff
@@ -3,47 +3,54 @@
 // Vendored from Uniswap core contracts with minor modifications:
 // - Added appropriate SPDX license comment
 // - Modified Solidity version
+// - Removed unused method
 // - Formatted code
 // - Shortened revert messages
 // - Converted to an abstract contract with virtual `pairFor` for testing
+// - Use interface types instead of opaque `address`es
```

Comment:
Is this just for convenience or does it have an impact on gas costs?

Reply:
Specifically, its because you can't cast `IERC20[]` to `address[]` and would require code to do a "reinterpret cast". Instead, I just `s/address/IERC20`.